core/templates/pages/exploration-player-page/services/content-translation-manager.service.ts

```diff
@@ -97,8 +100,9 @@ export class ContentTranslationManagerService {
       languageCode: string,
       content: SubtitledHtml
   ): string {
-    const writtenTranslation = writtenTranslations.translationsMapping[
-      content.contentId][languageCode];
+    const writtenTranslation = (
+      content.contentId ? writtenTranslations.translationsMapping[
```

Comment:
Should an be thrown instead ? Ditto below

Reply:
Why?